rbac/management/tenant_service/v2.py

```diff
@@ -0,0 +1,392 @@
+                partition_key="rbactodo",
+                remove=tuples_to_remove,
+            )
+        )
+
+    def _get_or_bootstrap_tenant(self, org_id: str, account_number: Optional[str] = None) -> BootstrappedTenant:
+        tenant_name = f"org{org_id}"
+        tenant, _ = Tenant.objects.get_or_create(
+            org_id=org_id,
+            defaults={"ready": True, "account_id": account_number, "tenant_name": tenant_name},
+        )
+        try:
+            mapping = TenantMapping.objects.get(tenant=tenant)
+            return BootstrappedTenant(
+                tenant=tenant,
+                mapping=mapping,
+            )
+        except TenantMapping.DoesNotExist:
+            bootstrap = self._bootstrap_tenant(tenant)
+            return bootstrap
+
+    def _bootstrap_tenant(self, tenant: Tenant) -> BootstrappedTenant:
+        # Set up workspace hierarchy for Tenant
+        root_workspace = Workspace.objects.create(
+            tenant=tenant,
+            type=Workspace.Types.ROOT,
+            name="Root Workspace",
+        )
+        default_workspace = Workspace.objects.create(
+            tenant=tenant,
+            type=Workspace.Types.DEFAULT,
+            parent=root_workspace,
+            name="Default Workspace",
+        )
+        tenant_id = f"{self._user_domain}:{tenant.org_id}"
+        relationships = [
+            create_relationship(
+                ("rbac", "workspace"),
+                str(default_workspace.uuid),
+                ("rbac", "workspace"),
+                str(root_workspace.uuid),
+                "parent",
+            ),
+            create_relationship(
+                ("rbac", "workspace"), str(root_workspace.uuid), ("rbac", "tenant"), tenant_id, "parent"
+            ),
+        ]
+
+        # Include platform for tenant
+        relationships.append(
+            create_relationship(("rbac", "tenant"), tenant_id, ("rbac", "platform"), settings.ENV_NAME, "platform")
+        )
+
+        mapping = TenantMapping.objects.create(tenant=tenant)
+        relationships.extend(self._bootstrap_default_access(tenant, mapping, default_workspace))
+
+        self._replicator.replicate(
+            ReplicationEvent(
+                event_type=ReplicationEventType.BOOTSTRAP_TENANT,
+                info={"org_id": tenant.org_id, "default_workspace_uuid": str(default_workspace.uuid)},
+                partition_key="rbactodo",
+                add=relationships,
+            )
+        )
+
+        return BootstrappedTenant(tenant, mapping)
+
+    def _bootstrap_default_access(
+        self, tenant: Tenant, mapping: TenantMapping, default_workspace: Workspace
+    ) -> List[Relationship]:
+        """
+        Bootstrap default access for a tenant's users and admins.
+
+        Creates role bindings between the tenant's default workspace, default groups, and system policies.
+        """
+        platform_default_role = self._get_platform_default_policy_uuid()
```

Comment:
Is intent here to return policy uuid or role uuid ? Could be name of variable be improved ?
for example `platform_default_[role|group]_uuid`
same for `admin_default_role`